blog/2017-09-25-streaming-to-another-database.adoc

```diff
@@ -0,0 +1,222 @@
+[source,bash,indent=0]
+----
+export DEBEZIUM_VERSION=0.6
+docker-compose up
+----
+
+When all components are started we are going to register JDBC Sink connector writing into PostgreSQL database
+[source,bash,indent=0]
+----
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @jdbc-sink.json
+----
+
+using this registration request.
+[source,json,indent=0]
+----
+{
+    "name": "jdbc-sink",
+    "config": {
+        "connector.class": "io.confluent.connect.jdbc.JdbcSinkConnector",
+        "tasks.max": "1",
+        "topics": "customers",
+        "connection.url": "jdbc:postgresql://postgres:5432/inventory?user=postgresuser&password=postgrespw",
+        "transforms": "unwrap",                                                  (1)
+        "transforms.unwrap.type": "io.debezium.transforms.UnwrapFromEnvelope",   (1)
+        "auto.create": "true",                                                   (2)
+        "insert.mode": "upsert",                                                 (3)
+        "pk.fields": "id",                                                       (4)
+        "pk.mode": "record_value"                                                (4)
+    }
+}
+----
+
+The request configures this options
+
+1. unwrapping Debezium complex format into a simple one
+2. automatically create target tables
+3. insert a row if it does not exist or update an existing one
+4. identify primary key stored in Kafka's record value field
+
+[source,bash,indent=0]
+----
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @source.json
+----
+The registration request contains SMT for routing records into topics with simple name.
+
+Let's check if the databases are synchronized.
+[source,bash,indent=0]
+----
+docker-compose exec mysql bash -c 'mysql -u $MYSQL_USER  -p$MYSQL_PASSWORD inventory -e "select * from customers"'
++------+------------+-----------+-----------------------+
+| id   | first_name | last_name | email                 |
++------+------------+-----------+-----------------------+
+| 1001 | Sally      | Thomas    | sally.thomas@acme.com |
+| 1002 | George     | Bailey    | gbailey@foobar.com    |
+| 1003 | Edward     | Walker    | ed@walker.com         |
+| 1004 | Anne       | Kretchmar | annek@noanswer.org    |
++------+------------+-----------+-----------------------+
+
+docker-compose exec postgres bash -c 'psql -U $POSTGRES_USER $POSTGRES_DB -c "select * from customers"'
+ last_name |  id  | first_name |         email         
+-----------+------+------------+-----------------------
+ Thomas    | 1001 | Sally      | sally.thomas@acme.com
+ Bailey    | 1002 | George     | gbailey@foobar.com
+ Walker    | 1003 | Edward     | ed@walker.com
+ Kretchmar | 1004 | Anne       | annek@noanswer.org
+----
+
+Next we try to add a new record and verify that it was replicated into target database.
```

Comment:
I think it's beneficial to state once again that we're going from MySQL to PostgreSQL, so I'd suggest changing this sentence:

> With the connectors still running, we can add a new row to the MySQL database and then check that it was replicated into the PostgreSQL database.